amethyst_rendy/src/camera.rs

```diff
@@ -395,3 +423,258 @@ impl<'a> PrefabData<'a> for ActiveCameraPrefab {
         Ok(())
     }
 }
+
+
+
+#[cfg(test)]
+mod tests {
+    // Our world-space is Y-up, X-right, z-away
+    // Thus eye-space is y-up, x-right, z-behind
```

Comment:
i don't get why eye-space (world-space relative to camera, right?) has inverted depth. Can you explain?